packages/react-composites/tests/browser/chat/app/index.tsx

```diff
@@ -64,7 +64,17 @@ function App(): JSX.Element {
                 )
               : undefined
           }
-          onRenderAvatar={customDataModel ? () => <text id="custom-data-model-avatar">Avatar</text> : undefined}
+          onFetchAvatarPersonaData={
+            customDataModel
+              ? (userId) =>
+                  new Promise((resolve) =>
+                    resolve({
+                      imageInitials: 'CI',
```

Comment:
nitty nit: CN for Custom Name?

Reply:
Intentional to show that imageInitial can be independent of the displayName

Reply:
got it ;)